### ISSUE TITLE
Add in the shielded reward rate table for phase 3 on the MASP and transfer modules

### ISSUE THREAD

**Issue by ChrisHoltDesign** [anoma/namada-interface#1334]
<img width="387" alt="Screenshot 2024-11-27 at 19 35 24" src="https://github.com/user-attachments/assets/048bdb22-d682-4195-9675-58e28bb9084e">

**Comment by brentstone** [comment]
@ChrisHoltDesign just want to clarify that this table should show the protocol annual inflation (w.r.t. NAM) for each of the incentivized assets. This is simple data that only changes by governance

**Comment by ChrisHoltDesign** [comment]
@brentstone Correct and or however best / are able to display some kind of shielded reward rates to users. 
We can always re-evaluate after phase 5 and or create the calculator tool at a later stage that was also discussed

**Comment by euharrison** [comment]
Should we add it on phase 3 or 4?

**Comment by brentstone** [comment]
@ChrisHoltDesign Marco and Mateusz are working on the shielded rewards expected in next MASP epoch (this cannot be expressed as a "rate" in particular)

**Comment by brentstone** [comment]
@Fraccaman @grarco @sug0 do we have the MASP token inflation rates in the indexer?

This info can be queried from an RPC via `namadac masp-reward-tokens` and then parse output (or see what SDK returns)

**Comment by grarco** [comment]
> [@Fraccaman](https://github.com/Fraccaman) [@grarco](https://github.com/grarco) [@sug0](https://github.com/sug0) do we have the MASP token inflation rates in the indexer?
> 
> This info can be queried from an RPC via `namadac masp-reward-tokens` and then parse output (or see what SDK returns)

Not yet, I need to start working on it. What is that we want exactly? I thought we wanted a reward estimation tool

**Comment by brentstone** [comment]
@grarco that is in a separate issue - this is different.

Each token has some annual inflation rate, i.e. Osmosis 1%, Cosmos 0.5%. We just want to display these numbers in a sidebar.

**Comment by brentstone** [comment]
Do we have a PR for this somewhere? @ChrisHoltDesign @jurevans

**Comment by ChrisHoltDesign** [comment]
Just a copy update. 'MASP Inflation rate'

<img width="286" alt="Image" src="https://github.com/user-attachments/assets/f9de56f9-eff4-44e9-b95a-55f56cc9804a" />
